axelrod/player.py

```diff
@@ -3,9 +3,10 @@
 import random
 import copy
 
-from axelrod.actions import Actions, flip_action
+from axelrod.actions import Actions, flip_action, Action
 from .game import DefaultGame
 
+from typing import Dict, Any
```

Comment:
It's not a big deal but usually standard library imports come before others. I guess we haven't discussed our official preference for typing imports.